zyspotify/__main__.py

```diff
@@ -14,16 +14,22 @@
 import logging
 from logging.handlers import RotatingFileHandler
 from logging.config import dictConfig
-
+from json import loads
 import errno
 
 try:
     __version__ = metadata.version("zyspotify")
 except metadata.PackageNotFoundError:
     __version__ = "unknown"
 
-_USERNAME = os.environ.get("USERNAME", None)
-_PASSWORD = os.environ.get("PASSWORD", None)
+_USERNAME = None
+_PASSWORD = None
+with open("/run/secrets/spotify_login") as login:
```

Comment:
Nitpick but can you use `pathlib` for this purpose. something like `json.loads(Path('path/filename').read_text()`